app/controllers/restaurants_controller.rb

```diff
@@ -13,6 +13,7 @@ def show
     @restaurant = Restaurant.find(params[:id])
     @reviews = @restaurant.reviews.order('created_at desc').page params[:page]
     @review = Review.new
+    @google_query = @restaurant.address.gsub(/\s/, "%20") + ",%20Boston,%20MA"
```

Comment:
You can generate the complete address first and then encode it at once with `CGI.encode`:

```ruby
CGI.encode(@restaurant.address + ", Boston, MA")
```

If might be worthwhile to add a method to return the full address:

```ruby
CGI.encode(@restaurant.full_address)
```